### DISCUSSION TITLE
TinyMCE - Rich text editor language configuration changes

### DISCUSSION THREAD

**Discussion by RobinJoe** [tinymce/tinymce#7320] [General]
The TinyMCE 5.9 release introduces improved language configuration. As a result the spellchecker_select_languages option has been superseded by the `content_langs` option.

The select languages option allows configuration of a list of languages you can select in a drop down menu from the toolbar above the WYSIWYG. The `content_langs` option expands on this ability, by introducing the ability to configure language codes following the Internet Engineering Task Force (IEFT) standards.

What are your thoughts on using the IEFT standards for the languages configuration?

**Comment by caoimhinsmo** [comment]
Yes to using ietf standard language codes.

It would be good, though, if TinyMCE took its default language from the textarea (or whatever) which it is embeded in.  I label the textarea with lang=$lang, where $lang is the language which the text is going to be in, which I always know in advance, and without TinyMCE the browser does spellchecking for that language as I type in the textarea. But when I use TinyMCE the spellcheck always defaults to English, even though I have specified “browser_spellcheck: true”.
